plugin/src/main/java/io/fabric8/maven/plugin/enricher/EnricherManager.java

```diff
@@ -54,6 +55,10 @@ public EnricherManager(EnricherContext enricherContext) {
         PluginServiceFactory<EnricherContext> pluginFactory = new PluginServiceFactory<>(enricherContext);
 
 
+        if (enricherContext.isUseProjectClasspath()) {
+            pluginFactory.addAdditionalClassLoader(ClassUtil.createProjectClassLoader(enricherContext.getProject(), enricherContext.getLog()));
```

Comment:
If we make the addition of the project classpath classloader unconditional then I would also add this class loader to the constructor of PluginServiceFactory to make the list of class loaders to use immutable after creation of the object.